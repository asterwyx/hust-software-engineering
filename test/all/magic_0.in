preset user AQ
step 63
1
dump