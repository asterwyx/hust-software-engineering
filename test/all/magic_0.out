user AQ
fund A 10000
fund Q 10000

credit A 0
credit Q 0

userloc A 63 0
userloc Q 0 1

nextuser A